packages/sdk/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Job } from './job';
+import { Pipeline } from './pipeline';
+export { JobStatus } from './utils';
+export { JobModel, PipelineModel, JobRunOption, PipelineInstallOption } from './interface';
+
+/**
+ * Pipcook daemon sdk for javascript
+ * the operations of pipeline and job is available for now.
+ */
+export class PipcookClient {
+  pipeline: Pipeline;
+  job: Job;
+
+  /**
+   * the constructor for PipcookClient
+   * @param host the daemon host, like 'http://192.168.1.50'
```

Comment:
Actually this is not as a host, it contains the protocol :(

Reply:
Yes, I have no idea about this, looks like there is no name for host with protocol.

Reply:
And `hostname` is also different from `host`:

```
host = hostname : port
```

Reply:
I think the simplest naming is to combine them: `schemaWithHostname` or `protocolWithHostname`.

Reply:
And `endpointPrefix` is reasonable, too.

Reply:
protocolWithHostname +1